### ISSUE TITLE
Quick bookmarks like qutebrowser?

### ISSUE THREAD

**Issue by sleeperzjf** [philc/vimium#3590]
I think the way that `qutebrowser `handles bookmarks is elegant.
Can't we do that with `vimium`?

**Comment by nerokeygen** [comment]
+1 for this. I came from qutebrowser because it's totally pain to emerge qtwebengine on gentoo specially with 8GB RAM that I can't use all threads.
but this is the only feature I miss.

**Comment by sleeperzjf** [comment]
Yes, if I open the bookmarks bar in my browser it looks ugly, if not, it's just painful to open certain bookmarks.

**Comment by sleeperzjf** [comment]
Or if I enable a customized search engine by typing in its shortcut and pressing `space`, then I just hit `enter`, the search string is empty and it just goes to that website.

**Comment by azinsharaf** [comment]
Hi, @sleeperzjf Have you found any solution to replicate `qutebrowser` bookmark? I'd like to use the bookmark file instead of using the bookmarks in the browser.
